src/features/serveStatic/ServeStatic.tsx

```diff
@@ -0,0 +1,55 @@
+import http, { Server } from 'http'
+import serveStatic from 'serve-static'
+import path from 'path'
+
+const servers: Server[] = []
+
+export default function initServeStatic(isPackaged: boolean): void {
+  let squooshStaticPath = `${process.cwd()}/node_modules/squoosh/production`
+  let glitchStaticPath = `${process.cwd()}/node_modules/jpg-glitch/production`
+  if (isPackaged) {
+    squooshStaticPath = path.join(
+      process.resourcesPath,
+      '/app.asar/.webpack/renderer/static/squoosh',
+    )
+    glitchStaticPath = path.join(
+      process.resourcesPath,
+      '/app.asar/.webpack/renderer/static/glitch',
+    )
+  }
+  setupServeStatic(squooshStaticPath, 5200)
+  setupServeStatic(glitchStaticPath, 5300)
```

Comment:
I think we can have the port as a constant in the feature folder of every tool, and call it here like `sqoosh.staticPort`, `glicth.staticPort`

Reply:
It has been updated